### ISSUE TITLE
unable to make developer build on Ubuntu 18.04.6 LTS

### ISSUE THREAD

**Issue by rkarthik2k1** [cisco/mlspp#266]
After cloning the repo, I was able to build with make (after installing OpenSSL). 
I also installed doctest in order to do a developer build using 'make dev' since there is dependency on doctest for the tests. However I am getting cmake errors about finding doctest: 

CMake Error at lib/bytes/test/CMakeLists.txt:14 (include):
  include could not find requested file:

    doctest

CMake Error at lib/bytes/test/CMakeLists.txt:16 (doctest_discover_tests):
  Unknown CMake command "doctest_discover_tests".

Looks like cmake is unable to find doctest. 

A solution for this could be
modify cmake configuration to enable finding doctest 
OR
add instruction steps to README consisting of simple commands to run

**Comment by bifurcation** [comment]
Hi @rkarthik2k1.  Sorry for not noticing this issue for a while.  You might need to install doctest:

https://www.howtoinstall.me/ubuntu/18-04/doctest-dev/

Closing since this is a local environment issue, not an MLSpp issue.

**Comment by rkarthik2k1** [comment]
Hi Richard Thanks for the response. Sorry I was on travel so I could not respond.  I have installed doctest on Linux and still face issues with cmake. I will try again but I think I was able to run the tests with PowerShell on windows.  I think I am ok since I am able to build at least on one platform.  Are there any documentations on how to use the libabry, any examples etc? If I take up any open issue I would like to test those Thanks Regards,Karthik From: Richard BarnesSent: Monday, July 25, 2022 1:10 AMTo: cisco/mlsppCc: Karthik Ramanathan; MentionSubject: Re: [cisco/mlspp] unable to make developer build on Ubuntu 18.04.6 LTS (Issue #266) Hi @rkarthik2k1. Sorry for not noticing this issue for a while. You might need to install doctest:https://www.howtoinstall.me/ubuntu/18-04/doctest-dev/Closing since this is a local environment issue, not an MLSpp issue.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
